### ISSUE TITLE
BERT: Pre-trainig of Deep Bidirectional Transformers for Language Understanding

### ISSUE THREAD

**Issue by hs-sugi** [One-Day-One-Paper/One-Day-One-Paper#165]
- Jacob Devlin, Ming-Wei Chang, Kenton Lee, Kristina Toutanova
- [Submitted on 11 Oct 2018 (v1), last revised 24 May 2019 (this version, v2)]

### Abstract
- 대용랑 unlabeled data로 모델을 미리 학습 시킨 후, 특정 task를 가지고 있는 labeled data로 transfer learning을 하는 모델
- pre-training 단계와 fine-tuning 단계, 두가지 단계로 구분
- pre-training
  - Task 1: Masked LM
    - 단어 중의 일부(비율 15%)를 [MASK] token 으로 바꿈.  
  - Task 2: Next Sentence prediction
    - corpus에서 두 문장을 이어 붙여 이것이 원래의 corpus에서 바로 이어 붙여져 있던 문장인지를 맞추는 binarized next sentence prediction task를 수행
    - 50% : sentence A, B가 실제 next sentence; 50% : sentence A, B가 corpus에서 random으로 뽑힌(관계가 없는) 두 문장
- 제작한 ppt(+발표대본포함) [BERTpart.pdf](https://github.com/One-Day-One-Paper/One-Day-One-Paper/files/8646615/BERTpart.pdf)
